signinapp/model.py

```diff
@@ -291,6 +303,33 @@ def get_from(cls, name: str, phone_number: str, email: str, contact_order: int)
         return guardian_user_data
 
 
+class Student(db.Model):
+    __tablename__ = "students"
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey("users.id"))
+
+    # Extra student information
+    graduation_year = db.Column(db.Integer)
```

Comment:
Should subteam be specific to students?

Reply:
I think it's worth storing for mentors as well? I suppose we could store it separately for both types?

Reply:
The fact that the majority of user types won't require it, only mentor and student (and lead technically, but that also has student data) makes me think it should live in those tables. Guardians, admin, display, etc all don't use it.

The downside is that it would affect the badge search and user list pages...

I would be fine delaying to 1.1, but if we do change it's a schema change that would move columns around so we have to decide now. If we want to do it later, we'd need it to be in 2.0

Reply:
Not going to make this change at this time